cmd/integration/integration.go

```diff
@@ -229,12 +303,24 @@ func startComponents(firstManifestURL, secondManifestURL, apiVersion string) (st
 	nodeController.Run(5*time.Second, true)
 	cadvisorInterface := new(cadvisor.Fake)
 
+	// Start the service account and service account token controllers
+	serviceaccount.NewTokensController(cl, serviceaccount.DefaultTokenControllerOptions(serviceaccount.JWTTokenGenerator(serviceAccountKey))).Run()
+	serviceaccount.NewServiceAccountsController(cl, serviceaccount.DefaultServiceAccountControllerOptions()).Run()
+
+	// Use normal directories for secret volume mounts, not tmpfs, to avoid needing to run with root permissions
+	secret.SetStorageMedium(api.StorageTypeDefault)
```

Comment:
Hrm, couldn't this also be accomplished by injecting a nop mounter into the kubelet?  I really dislike these kind of stateful package level constructs.